app/src/main/res/layout/splash_screen.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@color/n_0">
+
+    <ImageView
+        android:id="@+id/imageViewSplashScreen"
+        android:layout_width="228dp"
+        android:layout_height="118dp"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentBottom="true"
+        android:layout_marginStart="66dp"
```

Comment:
Using ConstraintLayout you will be able to avoid most of those hardcoded margins + keep in mind to use multiplies of 8dp so 195 -> 192 etc.